api/v1alpha1/openapi.yaml

```diff
@@ -3009,3 +3009,157 @@ components:
       - 'TemplateDiscriminatorKubernetesSec'
       - 'TemplateDiscriminatorInlineConfig'
       - 'TemplateDiscriminatorHttpConfig'
+    CertSigningRequestApproval:
```

Comment:
question: Approval sounds a bit presumptuous :) is this CertSigningRequestStatus?

Reply:
Hm, not sure what you mean - we have this schema for [EnrollmentRequests](https://github.com/flightctl/flightctl/blob/main/api/v1alpha1/openapi.yaml#L2478) and I figured it would work the same way. We've (me, Anton, and Frank) also talked about using "approvals" in the CSR policies before a certificate can be signed.